### PR TITLE
Added automatic test for coding standards

### PR DESCRIPTION
**Pull request by lsolesen**

### CODE REVIEW

**Comment by lsolesen** [comment]
Right now the pull reqeust does not check commerce.module:

@bojanz What do you want to do with `Drupal.Classes.FullyQualifiedNamespace.UseStatementMissing`

```
codio@total-patrol:~/workspace/motionsplan.dk/web/modules/contrib/commerce$ vendor/bin
/phpcs --standard=phpcs.xml commerce.module -s                                        

FILE: ...pace/motionsplan.dk/web/modules/contrib/commerce/commerce.module             
----------------------------------------------------------------------                
FOUND 2 ERRORS AFFECTING 2 LINES                                                      
----------------------------------------------------------------------                
 39 | ERROR | [x] Namespaced classes/interfaces/traits should be                      
    |       |     referenced with use statements                                      
    |       |     (Drupal.Classes.FullyQualifiedNamespace.UseStatementMissing)        
 54 | ERROR | [x] Namespaced classes/interfaces/traits should be                      
    |       |     referenced with use statements                                      
    |       |     (Drupal.Classes.FullyQualifiedNamespace.UseStatementMissing)        
----------------------------------------------------------------------                
PHPCBF CAN FIX THE 2 MARKED SNIFF VIOLATIONS AUTOMATICALLY                            
----------------------------------------------------------------------                

Time: 289ms; Memory: 6.75Mb                      
```

File: modules/order/tests/src/Kernel/Entity/OrderTest.php [hunk 1]
```diff
@@ -22,10 +23,11 @@ class OrderTest extends KernelTestBase {
    * @var array
    */
   public static $modules = ['system', 'field', 'options', 'user', 'entity',
```
Comment (bojanz) [inline modules/order/tests/src/Kernel/Entity/OrderTest.php]: We should move the first line as well then ("'system', 'field', 'options', 'user', 'entity',")

**Comment by bojanz** [comment]
Let's move those namespaces to use statements. I wanted to prevent needless autoloading (when the hooks aren't used), but these classes are generic enough and we should probaby move that logic to services anyway.

**Comment by lsolesen** [comment]
@bojanz Any exceptions for the coding standards that you want left in phpcs.xml?

**Comment by bojanz** [comment]
I think ClassComment.Missing and Drupal.Files.LineLength.TooLong are the only ones we need to disable

**Comment by lsolesen** [comment]
@bojanz What do you want to do about this one?

```
codio@total-patrol:~/workspace/motionsplan.dk/web/modules/contrib/commerce$ vendor/bin/phpcs --standard=phpcs.xml modules -s                                                                          

FILE: ...web/modules/contrib/commerce/modules/tax/src/TaxTypeImporter.php                          
----------------------------------------------------------------------                             
FOUND 0 ERRORS AND 2 WARNINGS AFFECTING 2 LINES                                                    
----------------------------------------------------------------------                             
  99 | WARNING | Only string literals should be passed to t() where                                
     |         | possible (Drupal.Semantics.FunctionT.NotLiteralString)                            
 120 | WARNING | Only string literals should be passed to t() where                                
     |         | possible (Drupal.Semantics.FunctionT.NotLiteralString)                            
----------------------------------------------------------------------                             

Time: 16.18 secs; Memory: 37.75Mb    
```

**Comment by lsolesen** [comment]
@bojanz And this one?

```
codio@total-patrol:~/workspace/motionsplan.dk/web/modules/contrib/commerce$ vendor/bin/phpcs --standard=phpcs.xml modules -s                                                                          

FILE: ...modules/contrib/commerce/modules/order/src/Form/OrderAddForm.php                          
----------------------------------------------------------------------                             
FOUND 1 ERROR AFFECTING 1 LINE                                                                     
----------------------------------------------------------------------                             
 54 | ERROR | Public method name "OrderAddForm::getFormID" is not in                               
    |       | lowerCamel format                                                                    
    |       | (Drupal.NamingConventions.ValidFunctionName.ScopeNotCamelCaps)                       
----------------------------------------------------------------------                             

Time: 16.12 secs; Memory: 37.75Mb
```

**Comment by bojanz** [comment]
getFormID should be getFormId.
Do what you must to the importer, it needs a full rewrite anyway.

Merged the current progress.

**Pull request closed without merging**